src/OSPSuite.Core/Chart/ChartSettings.cs

```diff
@@ -14,6 +14,25 @@ public enum LegendPositions
       BottomInside
    }
 
+   public enum GridGroupRowFormats
+   {
+      HideColumnName
+   }
+
+   public static class GridGroupRowFormatsExtensions
```

Comment:
Neat idea. Let's move this away from ChartSettings in the generic UI stuff 
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/tree/develop/src/OSPSuite.UI/Extensions

I would also create a constant for the default format in the enum (e.g. HideColumnName, Standard ) and put a comment with /// that will be shown on intellisense.
Then it would be cool to look for all the magic string in code and replace with this.

Last you can do it a bit differently also by creating your own enum on steroid. I
See https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/src/PKSim.Core/Model/ApplicationType.cs#L7

This is obviously less complicated but you can add as many properties to your enum as you want. Just food for thoughts